version.gradle.properties

```diff
@@ -1 +1 @@
-version=1.2.1
\ No newline at end of file
+version=1.2.2
```

Comment:
We do not update version manually, there is a script for that. Please revert this commit.